CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
+* Hide services cookie banner when JavaScrips is disabled ([PR #1586](https://github.com/alphagov/govuk_publishing_components/pull/1586))
```

Comment:
Why hide the banner for only services? The analytics on GOV.UK is also Javascript only, so it could be useful there as well.

Reply:
not really, on GOV.UK we use it for user settings too, some of them handled server-side. I won't touch the main GOV.UK one outside of a mission work 😬